src/modules/RA/Campaign/views/CampaignActions.js

```diff
@@ -37,22 +39,21 @@ const CampaignActions = ({
 
   return (
     <TopToolbar>
-
       {(data && data.id) && (
```

Comment:
```suggestion
      {data?.id && (
```

Reply:
I'm not even sure the optionnal chaining has an utility because data has an empty object as default value (see line.23)

Reply:
Very true!

Reply:
I change it